lib/jekyll-import/importers/wordpress.rb

```diff
@@ -129,9 +131,7 @@ def self.process(opts)
            WHERE posts.post_type = 'page'"
 
         db[page_name_query].each do |page|
-          if !page[:slug] || page[:slug].empty?
-            page[:slug] = sluggify(page[:title])
-          end
+          page[:slug] = sluggify(page[:title]) if !page[:slug] || page[:slug].empty?
```

Comment:
```ruby
page[:slug] = sluggify(page[:title]) if page.fetch(:slug, "").empty?
```

This is dependent upon `page` being a hash, of course. :smile:

Reply:
This doesn't break the tests.